notebooks/ch-mbqc/introduction.ipynb

```diff
@@ -0,0 +1,45 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "Most other chapters focus on a computation model where we describe our quantum algorithms and protocols as an array of gates, that is, a quantum circuit. However, there exists another representation for quantum operations in terms of an entangled resource state. The resource state is usually prepared in a particular way that makes it look like a cluster (2D fully conntected lattice). The measurement is performed sequentially on each qubit, one after another. Since the state is entangled, the remaining unmeasured qubits will undergo a side-effect of measuring a previous qubit accordingly. Additionally, after measuring a qubit, a few operations might be applied on the next qubits before measuring them (byproduct correction). \n",
```

Comment:
```suggestion
    "Most other chapters focus on a computation model where we describe our quantum algorithms and protocols as an array of gates, that is, a quantum circuit. However, there exists another representation for quantum operations in terms of an entangled resource state. The resource state is usually prepared in a particular way that makes it look like a cluster (2D fully connected lattice). The measurement is performed sequentially on each qubit, one after another. Since the state is entangled, the remaining unmeasured qubits will undergo a side-effect of measuring a previous qubit accordingly. Additionally, after measuring a qubit, a few operations might be applied on the next qubits before measuring them (byproduct correction). \n",
```